cms/templates/js/publish-xblock.underscore

```diff
@@ -97,10 +97,14 @@ var visibleToStaffOnly = visibilityState === 'staff_only';
                 <% } else { %>
                     <span class="icon fa fa-square-o" aria-hidden="true"></span>
                 <% } %>
-                <%- gettext('Hide from students') %>
+                <%- gettext('Hide from learners') %>
             </a>
             </li>
         </ul>
+        <p>
+            <%- gettext("Note: Do not hide graded assignments after they have been released.") %>
+            <a href="#"><%- gettext("Link to documentation.") %></a>
```

Comment:
@efischer19 I'm now thinking this link to doc isn't necessary. Can we remove? Thanks!